db/models.py

```diff
@@ -50,3 +53,65 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to="User",
+                             on_delete=models.CASCADE,
+                             null=True,
+                             blank=True,
+                             related_name="orders")
```

Comment:
The `Order` model uses a foreign key to a `User` model. Ensure that the `User` model is correctly defined and available in the application. If `User` is a custom model, make sure it is set as `AUTH_USER_MODEL` in your settings.